agent/fw_drupal8.c

```diff
@@ -529,6 +530,22 @@ NR_PHP_WRAPPER(nr_drupal8_name_the_wt_via_symfony) {
 }
 NR_PHP_WRAPPER_END
 
+void nr_drupal_version() {
+  char* string = "Drupal::VERSION;";
+  zval retval;
+  int result
+      = zend_eval_string(string, &retval, "Retrieve Drupal Version");
+  
+  // Add php package to transaction
+  if (result == SUCCESS) {
+    if (Z_TYPE(retval) == IS_STRING) {
+      char* version = Z_STRVAL(retval);
+      nr_txn_add_php_package(NRPRG(txn), "drupal/core", version);
+      zval_dtor(&retval);
+    }
```

Comment:
```suggestion
    }
    zval_dtor(&retval);
```
Even if the zval is somehow not a string, we still want to free it